### PR TITLE
data: rudra016

### PR DESCRIPTION
**Pull request by rudra016**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7377"></a>

### CODE REVIEW

File: data/rudra016.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      "title": "NetFlix Clone using ReactJS & Firebase",
+      "date": "April 2023",
+      "icon": "FaReact",
+      "color": "grey",
+      "description": "The Netflix clone website is built using ReactJS and Tailwind, User authentication is implemented using Firebase Authentication.",
+      "url": "https://netflix-clone-react-9a25e.web.app/"
+    }
+  ],
+  "displayStatsPublic": true,
+  "type": "personal",
```
Comment (krishstwt) [inline data/rudra016.json:36]: These fields are no longer needed, rest all is great thank you!
```suggestion
```

File: data/rudra016.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  "bio": "Aspiring Full-Stack Developer",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "ReactJS",
+    "NextJS",
+    "Java",
+    "FullStack"
+  ],
+  "avatar": "https://github.com/rudra016.png",
```
Comment (krishstwt) [inline data/rudra016.json:46]:
```suggestion
```

**Comment by rudra016** [comment]
@krishstwt now check it pls

**Review by krishstwt — APPROVED** [review]
Looks good!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**